aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -77,6 +74,11 @@ use aptos_types::{
         TransactionOutput, TransactionPayload, TransactionStatus, VMValidatorResult,
         ViewFunctionOutput, WriteSetPayload,
     },
+    vm::module_metadata::{
+        get_compilation_metadata_from_compiled_module,
```

Comment:
does it make sense to use the prelude for these?

Reply:
Added a prelude for these, excluding `get_metadata`, `get_metadata_v0`, and `verify_module_metadata` as these are VM only (+ in 1 place in indexer but for no good reason)